src/Common/Select.php

```diff
@@ -612,12 +612,14 @@ public function groupBy(array $spec)
      *
      * @param string $cond The HAVING condition.
      *
+     * @param array ...$bind arguments to bind to placeholders
```

Comment:
I was not suggesting to use php 7 string. It was just an example. But I don't have any trouble if we move to 7.0 as 5.6 has EOL . May be another issue :) .